net/librsync/Portfile

```diff
@@ -22,34 +25,16 @@ long_description    librsync is a free software library that implements \
                     wire-compatible with rsync 2.x, and is not likely to be \
                     in the future.
 
-homepage            http://librsync.sourceforge.net/
-master_sites        sourceforge:project/librsync/librsync/${version}
+homepage            http://librsync.sourceforg.net/
```

Comment:
Let's prefer to use `https://librsync.github.io`  as the `homepage`.

Please make links `https://` instead of `http://` everywhere you can.